.github/CODEOWNERS

```diff
@@ -123,7 +123,7 @@ pkg/experimentalmetricmetadata/                      @open-telemetry/collector-c
 processor/attributesprocessor/                       @open-telemetry/collector-contrib-approvers @boostchicken
 processor/cumulativetodeltaprocessor/                @open-telemetry/collector-contrib-approvers @TylerHelmuth
 processor/deltatorateprocessor/                      @open-telemetry/collector-contrib-approvers @Aneurysm9
-processor/filterprocessor/                           @open-telemetry/collector-contrib-approvers @boostchicken
+processor/filterprocessor/                           @open-telemetry/collector-contrib-approvers @boostchicken @TylerHelmuth
```

Comment:
Since you are an approver, add yourself first, so you are the "sponsor" as well :)